### ISSUE TITLE
Edit phone number pages - Accessibility issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16310]
## What Happened?
1. With VoiceOver enabled on iOS, the save button has multiple selections
2. With VoiceOver and TalkBack enabled on iOS and Android, the cancel button a11yHint is not being read
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

3. When a larger font/display size is enabled on iOS, the field labels are incorrectly formatted during resizing
4. Resizing of header content bug is covered in a [separate ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/15534)
     - NOTE: This affects the save button on iOS and Android

## Specs:
- Device: Android - LG LM-Q720 Stylo 5 
- Device: iOS - iPhone XR with VoiceOver enabled
- Bluetooth Keyboard

## Steps to Reproduce
For 1: 
1. With VoiceOver enabled on iOS, navigate to either of the edit phone number pages (home, work; etc)
2. Enter less than 10 numbers in the numbers field or at least 1 number in the extension field. Note that the save button becomes disabled
3. Press the save button. Note that the VoiceOver focus is only on the save text and VoiceOver only reads "save"
4.  Press the save button again. Note that the VoiceOver focus expands and VoiceOver reads "Save. Dimmed. Button. Saves Changes"

For 3: 
1. On iOS enable a larger font/display size
2. Navigate to either of the edit phone number pages (home, work; etc)
3. Note field labels are incorrectly formatted (cut off by letter)

## Desired Behavior
For 1:
- With VoiceOver enabled on iOS, the save button will have one selection state

For 3:
- On iOS, when a larger font/display size is enabled, the field label will not be cut off by letter 
     - Note: Will have to recheck expected behavior

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 3 - Medium

## Linked Story
- Story #13894

## Screen Shot (s) and Additional Information
6. iOS field label resizing bug 🔽
![Edit Phone Number - Field Label.PNG](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/c1e2eacd-6485-472e-a99e-7fd0d6fa8916)

7. Resizing bug is covered in a [separate ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/15534). Included how it affects edit phone number pages
- Android view of bug 🔽
![Edit Phone Number - Android Save Button.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/5c2256d7-ee58-4b08-98f7-ac9598d55dd0)
- iOS view of bug 🔽
![Edit Phone Number - iOS Save Button.PNG](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/fae85bcf-9191-449b-8af6-026be10d0ce2)

**Comment by digitalken** [comment]
PASS 1/14/2021 - Looks good on both iOS and Android.  Closing.
